client/src/components/Tables/Resource/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { FC, ReactElement, ReactNode } from 'react';
+import { Icon } from 'semantic-ui-react';
+import { IResource } from 'ssw-common';
+
+import { FieldTag } from '../..';
+import DynamicTable from '../DynamicTable';
+import { buildColumn } from '../DynamicTable/util';
+
+interface ResourceTableProps {
+  resource: IResource[];
+  handleOpen: (selected: IResource) => void;
+  isAdmin: boolean;
+}
+
+const ResourceTable: FC<ResourceTableProps> = ({
+  resource,
+  handleOpen,
+  isAdmin,
+}): ReactElement => {
+  const titleColumn = buildColumn<IResource>({
+    title: 'Title',
+    width: 16,
+    sorter: (a: IResource, b: IResource) => a.name.localeCompare(b.name),
+    extractor: function getTitle(resource: IResource): ReactNode {
+      return resource.name;
+    },
```

Comment:
You can actually just specify the extractor as the string 'name' since it's a valid key of `IResource`, as seen below:
```suggestion
    extractor: 'name',
```
The DynamicTable takes care of the rest.